llvm/lib/Target/RISCV/MCTargetDesc/RISCVAsmBackend.h

```diff
@@ -47,9 +51,15 @@ class RISCVAsmBackend : public MCAsmBackend {
   bool evaluateTargetFixup(const MCFixup &Fixup, const MCValue &Target,
                            uint64_t &Value) override;
 
+  std::optional<StringRef>
+  getVendorIdentifierForFixup(unsigned TargetFixupKind) const;
```

Comment:
rename this to `FixupKind` to be consistent with the impl.

(Beside FK_Data_ fixup kinds are almost always target-specific. "Target" probably does not convey extra meaning. )